src/ToDo.Tests/Services/TaskEndpointTests.cs

```diff
@@ -51,8 +51,8 @@ internal class TaskEndpointTests : BaseEndpointTests<ITaskEndpoint>
 	public async System.Threading.Tasks.Task Create_TodoTask_ShouldReturn_NewTask()
 	{
 		//Arrange
-		var listId = "AAMkAGFlMTMyOTVlLTg4MTYtNGNkYi05Y2I1LWIxNjQ3MjQzZGUwZgAuAAAAAABxiwJ7rbfvTL0IfGDSJ4lUAQAstIhkSEopRrR__AvQNI34AACzQA1BAAA=";
-		var newTask = new TaskData { Title = "new task" };
+		var listId = "AAMkAGM0ZTZiY2IwLTliZWEtNDM5Zi1iMDBlLTUxZDQxNWNmY2IxNgAuAAAAAAAC8Egk03A8QrAy_y5u1QQAAQD-PT2STVFATpxIXsYfLHGvAADbMaF-AAA=";
```

Comment:
Can we create a separate test user account (eg unotodoapp_testuser@outlook.com) that we can populate with data so that we can all run these tests without having to change the listId

Reply:
Of course

Reply:
This isn't required for this PR but if you can do this and update these test ids, that would be great